packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js

```diff
@@ -87,11 +88,11 @@ class DateTimeView extends React.PureComponent {
 			leftElement: (
 				<Action
 					aria-label={t('DATEPICKER_MONTH_PREVIOUS', { defaultValue: 'Go to previous month' })}
-					icon="talend-chevron-left"
+					icon="talend-arrow-left"
 					label=""
 					onClick={() => this.goToPreviousMonth()}
 					tabIndex="-1"
-					link
+					className={classNames('btn-tertiary', 'btn-info')}
```

Comment:
The left arrow is not described as a tertiary button in guideline

Reply:
Hi, The left/right arrow is tertiary button in guideline.